recipes/hdfs3/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: hdfs3
```

Comment:
`hdfs3` is a Python wrapper to the `libhdfs3` C++ library, so we named it accordingly. `python-hdfs3` might add more confusion to the namespace of HDFS-related libraries.